core/app/modules/kit-library/module.php

```diff
@@ -0,0 +1,90 @@
+<?php
+namespace Elementor\Core\App\Modules\KitLibrary;
+
+use Elementor\Plugin;
+use Elementor\TemplateLibrary\Source_Local;
+use Elementor\Core\Base\Module as BaseModule;
+use Elementor\Core\Common\Modules\Connect\Module as ConnectModule;
+use Elementor\Core\Common\Modules\Connect\Apps\Library;
+use Elementor\Core\App\Modules\KitLibrary\Data\Api_Client;
+use Elementor\Core\App\Modules\KitLibrary\Data\Repository;
+use Elementor\Core\App\Modules\KitLibrary\Data\Kits_Controller;
+use Elementor\Core\App\Modules\KitLibrary\Data\Taxonomies_Controller;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+class Module extends BaseModule {
+	/**
+	 * Get name.
+	 *
+	 * @access public
+	 *
+	 * @return string
+	 */
+	public function get_name() {
+		return 'kit-library';
+	}
+
+	/**
+	 * Register the admin menu.
+	 */
+	private function register_admin_menu() {
+		add_submenu_page(
+			Source_Local::ADMIN_MENU_SLUG,
+			__( 'Kit Library', 'elementor' ),
+			__( 'Kit Library', 'elementor' ),
+			'manage_options',
+			Plugin::$instance->app->get_base_url() . '#/kit-library'
+		);
+	}
+
+	private function set_kit_library_settings() {
+		if ( ! Plugin::$instance->common ) {
+			return;
+		}
+
+		/** @var ConnectModule $connect */
+		$connect = Plugin::$instance->common->get_component( 'connect' );
+
+		/** @var Library $library */
+		$library = Plugin::$instance->common->get_component( 'connect' )->get_app( 'library' );
+
+		Plugin::$instance->app->set_settings( 'kit-library', [
+			'subscription_plans' => $connect->get_subscription_plans( 'kit-library' ),
+			'is_pro' => false,
+			'is_library_connected' => $library->is_connected(),
+			'library_connect_url'  => $library->get_admin_url( 'authorize' ),
+			'access_level' => ConnectModule::ACCESS_LEVEL_CORE,
```

Comment:
maybe `base_access_level`?

Reply:
This is the "access_level" of the current license and not the "base access level". it will be replace by the "pro"